hack/run-linter.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd $(dirname $0)/..
+
+source hack/lib.sh
+
+CONTAINERIZE_IMAGE=golang:1.18.4 containerize  ./hack/run-linter.sh
+
+go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
```

Comment:
Use v1.46.2 like here: https://github.com/pluralsh/plural-cli/blob/master/.github/workflows/ci.yaml#L54

Reply:
There is official Docker image that you can use: https://golangci-lint.run/usage/install/#local-installation